src/__experimental__/components/select/select.component.js

```diff
@@ -32,7 +33,20 @@ class Select extends React.Component {
     this.listboxId = uniqueId('listbox-');
   }
 
```

Comment:
Not a blocking comment but I think that if we already define state and isComponentControlled class fields outside of constructor then listboxId could be initialized in the same way so we would get rid of constructor. Or the other way around, define state and isComponentControlled inside of a constructor for consistency.

Reply:
Good point. Nice to remove the constructor! Refactored in cc68577 :thumbsup: